templates/settings.py.erb

```diff
@@ -1,8 +1,6 @@
-CONTENT_HOST = "<%= @fqdn %>"
-ANSIBLE_API_HOSTNAME = "<%= @fqdn %>"
-ANSIBLE_CONTENT_HOSTNAME = "https://<%= @fqdn %>/pulp_ansible/content/"
+CONTENT_HOST = "<%= scope['pulpcore::servername'] %>"
```

Comment:
Just a nit: can we get `CONTENT_HOST` and `CONTENT_ORIGIN` together?

Reply:
sure, this makes sense and I'm happy to do it